src/recipes/fasion-lemonade.json

```diff
@@ -0,0 +1,18 @@
+{
+    "name": "Banana Smoothie",
```

Comment:
Wrong recipe.

Reply:
i think i delete this on new commit, it wrong recipe